reveal-js/common.css

```diff
@@ -18,6 +18,10 @@ section > *:not(section) {
   overflow: hidden;
 }
 
+.slides p.big {
```

Comment:
please move it to the develop branch and add usage examples
new slides in the template html

Reply:
> please move it to the develop branch and add usage examples new slides in the template html

it seems that I don't have right to push code in the develop branch directly. @Dzmitry-Klokau could you help with this?

Reply:
done